test/integration/parser/facebook_item_test.rb

```diff
@@ -12,7 +12,6 @@ class FacebookItemIntegrationTest < ActiveSupport::TestCase
     assert !data['title'].blank?
     assert !data['username'].blank?
     assert !data['author_name'].blank?
-    assert !data['author_picture'].blank?
```

Comment:
Apify is no longer returning the Author's image in it's response, just the name and id. We have a [ticket](https://meedan.atlassian.net/browse/CV2-5858) for the resulting error.